semantic_conventions/trace/database.yaml

```
@@ -294,8 +294,8 @@ groups:
       - id: table
         type: string
         tag: call-level-tech-specific-cassandra
-        required:
-          conditional: Recommended if available.
+        requirement_level:
+          recommended: When available.
```

IMHO we can simplify this to

```suggestion
        requirement_level: recommended
```

since this is the default for "recommended".

(and other locations)